entrypoint.sh

```diff
@@ -21,6 +21,7 @@ then
   return -1
 fi
 
+INPUT_DESTINATION_HEAD_BRANCH_PREFIX="$INPUT_DESTINATION_HEAD_BRANCH"
```

Comment:
Does it make sense to just rename this variable or just use `INPUT_DESTINATION_HEAD_BRANCH` in the line below?

Reply:
`INPUT_DESTINATION_HEAD_BRANCH` may or may not have a timestamp appended to it (but does in our usage). This means that the search would fail since we want to search based on `automatic-styles-sync` or whatever that base value is.